glove2word2vec.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Manas Ranjan Kar <manasrkar91@gmail.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+CLI USAGE: python glove2word2vec.py -input <GloVe vector file> -output <Output model file>
+
+Convert GloVe vectors into word2vec C format;
+
+model = gensim.models.Word2Vec.load_word2vec_format('/tmp/vectors.txt', binary=False)  # C text format
+
+word2vec embeddings start with a line with the number of lines (tokens?) and the number of dimensions of the file. This allows gensim to allocate memory 
+accordingly for querying the model. Larger dimensions mean larger memory is held captive. Accordingly, this line has to be inserted into the GloVe 
+embeddings file.
+"""
+
+
+import re
+import sys
+import gensim
+import logging
+import argparse
+import smart_open
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+def glove2word2vec(glove_vector_file, output_model_file):
+    """Convert GloVe vectors into word2vec C format"""
+    
+    def count_dims(filename):
+        """ 
+        Function to calculate the number of dimensions from an embeddings file
+        """
+        count=0
+        dims=[]
+        for line in smart_open.smart_open(filename):
+            count+=1
```

Comment:
PEP8: spaces around operators (here and a lot of other places).

@manasRK , does your editor support highlighting PEP8 issues? Installing a Python linter should prevent such formatting problems.

It's not a big deal, we can reformat ourselves, but this may be helpful to you in the future, when you program in Python.